clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -62,8 +58,7 @@ class ServerService private (
   private[service] def startup(): Future[Unit] = {
     for {
       _ <- serverStatusDAO.setStatus(ServerStatusInfo.Starting)
-      _ <- waitForElasticsearchReady()
-      _ <- createOrUpdateIndices()
+      _ <- searchDAO.initialize()
```

Comment:
The ES server must already be running, but may be empty. `initialize()` will create or update the mappings. Meanwhile the akka-http server `startup()` actually starts listening for connections to the server.